packages/env/lib/cache.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const fs = require( 'fs' ).promises;
+
+/**
+ * Options for cache parsing.
+ *
+ * @typedef WPEnvCacheOptions
+ * @property {string}  workDirectoryPath Path to the work directory located in ~/.wp-env.
+ */
+
+const CACHE_FILE_NAME = 'wp-env-cache.json';
+
+/**
+ * This function can be used to compare a possible new cache value against an
+ * existing cache value. For example, we can use this to check if the configuration
+ * has changed in a new run of wp-env start.
+ *
+ * @param {string}           key     A unique identifier for the cache.
+ * @param {any}              value   The value to check against the existing cache.
+ * @param {WPEnvCacheOptions} options Parsing options
+ *
+ * @return {boolean} If true, the value is different from the cache which exists.
+ */
+async function didCacheChange( key, value, options ) {
+	const existingValue = await getCache( key, options );
+	return value !== existingValue;
+}
+
+/**
+ * This function persists the given cache value to the cache file. It creates the
+ * file if it does not exist yet, and overwrites the existing cache value for the
+ * given key if it already exists.
+ *
+ * @param {string}           key     A unique identifier for the cache.
+ * @param {any}              value   The value to persist.
+ * @param {WPEnvCacheOptions} options Parsing options
+ */
+async function setCache( key, value, options ) {
+	const existingCache = await getCacheFile( options );
+	existingCache[ key ] = value;
+
+	await fs.writeFile(
+		getPathToCacheFile( options.workDirectoryPath ),
+		JSON.stringify( existingCache )
+	);
+}
+
+/**
+ * This function retrieves the cache associated with the given key from the file.
+ * Returns undefined if the key does not exist or if the cache file has not been
+ * created yet.
+ *
+ * @param {string}           key     The unique identifier for the cache value.
+ * @param {WPEnvCacheOptions} options Parsing options
```

Comment:
Alignment

Reply:
👍 makes sense. when i renamed from `meta` to `cache` it knocked it out of alignment